app/controllers/spree/user_sessions_controller.rb

```diff
@@ -24,14 +23,10 @@ def create
       if spree_user_signed_in?
         flash[:success] = t('devise.success.logged_in_succesfully')
 
-        render cable_ready: cable_car.redirect_to(
-          url: return_url_or_default(after_sign_in_path_for(spree_current_user))
-        )
+        redirect_to after_sign_in_path_for(spree_current_user)
```

Comment:
Any reason to remove `return_url_or_default`? This was the only usage of that method.

`±` I think it was here for a reason (It returns you to where you were before you tried logging in), so can you add it back?